daemon/src/main/scala/io/gearpump/cluster/master/AppManager.scala

```diff
@@ -214,6 +213,12 @@ private[cluster] class AppManager(kvService: ActorRef, launcher: AppMasterLaunch
         case AppMasterNonExist =>
           sender ! AppMasterData(AppMasterNonExist)
       }
+
+    case uploadTasks: UploadTaskJar =>
+      uploadTasks.tasks.foreach(uploadedTasks += _ -> uploadTasks.jar)
```

Comment:
It doesn't make any sense to create another mapping uploadTasks....

As ProcessorDescription already contains all the information.
in ProcessorDescription, there is taskClass name, and AppJar.